libs/librepcb/common/graphics/graphicsview.cpp

```diff
@@ -267,7 +267,8 @@ bool GraphicsView::eventFilter(QObject* obj, QEvent* event) {
       if (e->button() == Qt::MiddleButton) {
         mCursorBeforePanning = cursor();
         setCursor(Qt::ClosedHandCursor);
-      } else if (mEventHandlerObject) {
+      }
+      if (mEventHandlerObject) {
```

Comment:
The panning action seemed like a click and hold action, that is different from a click. At one point I had a test case to toggle between top and bottom layer with a short click.

In my opinion there is a difference to the user between the universal pan action and a more specific tool dependent action. And the code shouldn't restrict that. With this change it is now necessary for the maintainers to spot changes, that produce conflicting behavior.

Reply:
OK I don't understand 100% what you mean, I still think it could lead to strange behavior if both, the GraphicsView and the editor tool reacts on the same event. But at least currently there is no such strange behavior noticeable, so let's keep it :wink: